README.md

```diff
@@ -1,60 +1,66 @@
+conda env create -n rotary --file=rotary/enviroment.yaml
 
 conda activate rotary
 
-mkdir -p output_dir conda_envs
-
-snakemake --snakefile rotary/rules/rotary.smk \
-  --configfile myconfig.yaml \
-  --directory output_dir \
-  --conda-prefix conda_envs \
-  --jobs 20 \
-  --use-conda \
-  --conda-frontend mamba \
-  --rerun-incomplete \
-  --reason \
-  --printshellcmds 2>&1 | \
-  tee rotary.log
+cd rotary
+
+pip install --editable .
+```
+
+### Run One Sample
+```bash
+mkdir output_dir
+mkdir rotary_db_dir
+
+cd output_dir
+
+rotary run_one -l s1_long.fastq.gz -r1 s1_R1.fastq.gz -r2 s1_R2.fastq.gz -d ../rotary_db_dir
 ```
 
 ## Description
-_rotary_ is a snakemake pipeline that can be used to assemble single microbial genomes using 
-standlone Nanopore data<sup>[1](#Footnotes)</sup> or hybrid Nanopore + short read data. 
+
+_rotary_ is a snakemake pipeline that can be used to assemble single microbial genomes using
+standlone Nanopore data<sup>[1](#Footnotes)</sup> or hybrid Nanopore + short read data.
 The pipeline performs long read QC, assembly, end repair, polishing, contig rotation, and genome annotation.
 
 ### Some advantages of using _rotary_:
+
 - All databases auto-install, so you can start analyzing genomes reproducibly with limited effort!
 - Snakemake checkpointing allows you to re-start a failed run from where you left off
-- Circularization is handled fairly carefully. Unlike the defaults in most pipelines, _rotary_ fixes the 
-  [short gap region](https://github.com/fenderglass/Flye/issues/315#issuecomment-720679812) that can occur at the ends 
-  of circular contigs produced by Flye. It also polishes the circular contigs in two 
+- Circularization is handled fairly carefully. Unlike the defaults in most pipelines, _rotary_ fixes the
+  [short gap region](https://github.com/fenderglass/Flye/issues/315#issuecomment-720679812) that can occur at the ends
+  of circular contigs produced by Flye. It also polishes the circular contigs in two
   different rotation states to try to correct errors near contig ends.
 
 ## Requirements
+
 - OS: Runs on Linux (tested on Ubuntu 20.04)
 - Software: requires `miniconda`
 - Resources: The majority of the pipeline is not too resource intensive. The limiting factors are:
-  - Flye requires moderate RAM (e.g., < 64 GB for typical bacterial genome runs)
-  - GTDB-Tk v2 (with GTDB r207) needs ~55 GB RAM
-  - EggNOG-mapper is a bit slow (e.g., ~40 minutes on 40 CPU threads)
+    - Flye requires moderate RAM (e.g., < 64 GB for typical bacterial genome runs)
+    - GTDB-Tk v2 (with GTDB r207) needs ~55 GB RAM
+    - EggNOG-mapper is a bit slow (e.g., ~40 minutes on 40 CPU threads)
 
 ## Usage
+
 ### 1. Install
+
 ```bash
 git clone https://github.com/jmtsuji/rotary.git
-conda env create -n rotary --file=rotary/envs/rotary.yaml
+conda env create -n rotary --file=rotary/envs/enviroment.yaml
 ```
 
 ### 2. Copy and fill out the config (YAML) file (`config.yaml`).
```

Comment:
We'll need to update this section later -- I assume `rotary init` now generates the config file?
I think the README can be further updated in future before pushing to master and does not need to be fully updated before this PR is merged to develop.

Reply:
I agree that the readme needs to be updated before the final release. Yes, `rotary init` is the as `atlas init`. It makes the `samples.tsv` and the `config.yaml` and places them in the project directory. You can point `rotary init` at a folder of fastq files.

Reply:
Added as issue.